specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2021-10-01/remediations.json

```diff
@@ -1078,6 +1078,10 @@
           "items": {
             "$ref": "#/definitions/RemediationDeployment"
           },
+          "x-ms-identifiers": [
+            "remediatedResourceId",
```

Comment:
>remediatedResourceId [](http://example.com/codeflow?start=13&length=20)

just remediatedResourceId is fine, each remediation only touches the same resource once